subprojects/core/src/main/java/org/gradle/initialization/IGradlePropertiesLoader.java

```diff
@@ -15,20 +15,16 @@
  */
 package org.gradle.initialization;
 
-import org.gradle.api.internal.properties.GradleProperties;
+import org.gradle.initialization.properties.MutableGradleProperties;
 
 import java.io.File;
 
 public interface IGradlePropertiesLoader {
 
-    String SYSTEM_PROJECT_PROPERTIES_PREFIX = "org.gradle.project.";
-
-    String ENV_PROJECT_PROPERTIES_PREFIX = "ORG_GRADLE_PROJECT_";
-
```

Comment:
I'd suggest to keep these properties and not the copy in `ProjectPropertiesLoader`, as these are actually used in third-party plugins.